MANIFEST.in

```diff
@@ -1,15 +0,0 @@
-include *.yml
-include LICENSE
-include MANIFEST.in
-include pyproject.toml
```

Comment:
Metadata is already included in wheels when built, so all this would do is export our linter configurations